packages/subgraph/src/packages/token/governance-erc20.ts

```diff
@@ -35,3 +42,44 @@ export function handleTransfer(event: Transfer): void {
     toMember.save();
   }
 }
+
+export function handleDelegateChanged(event: DelegateChanged): void {
+  let context = dataSource.context();
+  let pluginId = context.getString('pluginId');
+
+  // make sure `fromDelegate` &  `toDelegate`are members
+  if (event.params.fromDelegate != Address.zero()) {
+    let fromMember = getOrCreateMember(event.params.fromDelegate, pluginId);
+    fromMember.save();
+  }
+  if (event.params.toDelegate != Address.zero()) {
+    let toMember = getOrCreateMember(event.params.toDelegate, pluginId);
+    toMember.save();
+  }
+
+  // make sure `delegator` is member and set delegatee
+  if (event.params.delegator != Address.zero()) {
+    let delegator = getOrCreateMember(event.params.delegator, pluginId);
+
+    // set delegatee
+    let delegatee = event.params.toDelegate
+      .toHexString()
+      .concat('_')
+      .concat(pluginId);
+
+    delegator.delegatee = delegatee;
+
+    delegator.save();
+  }
+}
+
+export function handleDelegateVotesChanged(event: DelegateVotesChanged): void {
+  let context = dataSource.context();
+  let pluginId = context.getString('pluginId');
+
+  if (event.params.delegate != Address.zero()) {
+    let member = getOrCreateMember(event.params.delegate, pluginId);
+    member.votingPower = event.params.newBalance;
```

Comment:
I understand that this `newBalance` is the total number no?. I mean the own owned tokens + the sum of all the delegated one

Reply:
yes, it is computed by the contract

Reply:
It is. The field `votingPower` should have a comment to clarify

Reply:
amended.